app-api-common/recipe/src/main/java/org/codeforvictory/android/recipes/data/api/recipe/RecipeServiceProvider.kt

```diff
@@ -0,0 +1,17 @@
+package org.codeforvictory.android.recipes.data.api.recipe
+
+import retrofit2.Retrofit
+
+internal class RecipeServiceProvider(private val retrofit: Retrofit = RetrofitProvider.value) {
+
+    private var singleton: RecipeService? = null
```

Comment:
The property is a new one every time we create a new RecipeServiceProvider, right? Shouldn't we make it equivalent to static to be a singleton?

Reply:
Maybe object?

Reply:
Addressed: [`4606ac9` (#13)](https://github.com/CodeForVictory/recipes-mvi-playground/pull/13/commits/4606ac9ce1e6d7155cba6ed2f1f3145ed6ee3929)